content/applications/services/field_service/managing_products.rst

```diff
@@ -0,0 +1,44 @@
+==================================
+Managing products in Field Service
+==================================
+
+When field service workers perform on-site, they commonly use products for the service. Odoo’s
+**Field Service** app allows them to record the products they’re using. Doing so keeps your
+inventory up-to-date in real-time and automatically adds these products to the invoice.
+
+Add products to a task
+======================
+
+#. Go to :guilabel:`Field Service`
+#. Select a task
+#. Click the :guilabel:`Products` Smart Button
+#. Click a product card to add it to your task. You can click anywhere on the card to add the
```

Comment:
I agree it's kinda weird to phrase it like that. I've made this suggestion instead because if the devs one day decide to change the interface, my bet is that they will keep the Add button. What do you think?

```suggestion
#. Click :guilabel:`Add` on a product card to add it to your task.
```